lib/modules/device/DevicesController.ts

```diff
@@ -514,14 +514,15 @@ export class DevicesController {
         const exportId = request.getString("exportId");
 
         const { id } = await this.measureExporter.getExport(engineId, exportId);
+        const filename = request.getString("filename") || `device-${id}.csv`;
```

Comment:
I feel like we should enforce the .csv on our end and not let the user decide what extension is being used here to avoid issues.

Reply:
@rolljee 
I think this depends on how UI communicates with users. we can go even further by detecting filename to check if it contains extension also checking for extension. I'd update PR if that's the requirement.